codebusters-ca/firebase-course
# update mentorship instruction

This pr updates the following.

- Updated firebase setup instructions in README to ensure clarity and accuracy for users.
- Removed the project id from the run emulator script in package.json to enhance security.
- Added a firebase setup script for the project.
- Removed ``` Change admin status``` section for test mentee 
- Removed ```Functions``` folder and cloud function related setup
- Added a Prettier configuration to standardize the coding style for consistent, readable code.
- Reformat all code with ```Prettier```


Closes #17 

<img width="822" alt="Screen Shot 2023-03-07 at 16 50 08" src="https://user-images.githubusercontent.com/28695185/223563794-196cce7e-0682-41cd-84c7-8e2b171bc522.png">

<img width="777" alt="Screen Shot 2023-03-07 at 15 58 39" src="https://user-images.githubusercontent.com/28695185/223559410-7d43faa2-9a34-4570-9426-1ce1ade5e60d.png">

## Testing Steps / QA Criteria

- Follow [README](https://github.com/codebusters-ca/firebase-course/tree/update-mentorship-2023#readme) to set up a test firebase project 
- To run Firebase Emulators: npm run emulators
- To run the app locally: npm run dev. The app will run on [localhost:3000/](http://localhost:3000/)
- Test the app to make sure everything works as it should. 
- To run the tests, run the emulator by ``` npm run emulators```, then open another terminal run ```npm test```
